.github/workflows/Publish.yml

```
@@ -16,4 +16,5 @@ jobs:
       - run: npm install
       - run: npm publish --access public
         env:
+          # Access token for NPM registry for @qwerty287's account.
```

No, I meant I comment, where the access token is configured. In case it stops working in the future, nobody wants to search for it.
```suggestion
          # (Name, value)-pairs for access tokens are configured at https://github.com/LycheeOrg/basicContext/settings/secrets/actions
```

---

Oh OK :laughing:

---

> nobody wants to search for it.

Where does that stop? Do we need to point to this file in our readme so that we know where the actions are defined?